python/lammps.py

```diff
@@ -85,14 +85,19 @@ def __init__(self,name="",cmdargs=None,ptr=None,comm=None):
     # fall back to loading with a relative path,
     #   typically requires LD_LIBRARY_PATH to be set appropriately
 
+    if 'liblammps.dylib' in os.listdir(modpath):
```

Comment:
Please note, that this test only works for `name == ""`.

Reply:
I updated the condition to use: 
`any([f.startswith('liblammps') and f.endswith('.dylib') for f in os.listdir(modpath)])`
instead.

Reply:
@jan-janssen thanks. since your pull request is pointing out a somewhat fundamental issue, we will have to make some additional tests on a MacOS machine and also look into keeping changes consistent across the two supported build systems. i've read up a little on the topic of shared libraries and shared objects on MacOS and have now remote access to a MacOS machine. i will look into it later this week or next week, when i expect to have some larger blocks of free time. thanks again for bringing this to our attention and your patience and perseverance in keeping this in sync with all our little concerns and nitpicks.